new_doc/static/img/typeorm-logo-colored-dark.svg

```diff
@@ -0,0 +1,5 @@
+<svg width="581" height="115" viewBox="0 0 581 115" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Maybe worth adding to resources folder in the root and symlink to it but not crucial

Reply:
I see, let's create another PR for this. Thanks